src/io/io.go

```diff
@@ -378,6 +378,10 @@ func CopyN(dst Writer, src Reader, n int64) (written int64, err error) {
 // the copy is implemented by calling src.WriteTo(dst).
 // Otherwise, if dst implements the ReaderFrom interface,
 // the copy is implemented by calling dst.ReadFrom(src).
+// Copy returns the same error as the WriterTo and
+// ReaderFrom encounter. in most case
+// not return EOF error in the implenet of WriterTo
+// and ReaderFrom.
```

Comment:
This is not grammatically correct, and I feel that this documentation should actually be added to the `io.WriterTo` and `io.ReaderFrom` interface definitions, not `Copy` -- because people are going to read the `WriterTo` documentation when implementing `WriteTo`, not the documentation for `Copy`.